ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/435__data-science-pipelines-elyra.robot

```diff
@@ -0,0 +1,153 @@
+${SVG_PIPELINE_NODES} =    /*[name()="g" and @class="d3-nodes-links-group"]
+${SVG_SINGLE_NODE} =    /*[name()="g" and contains(@class, "d3-draggable")]
+${PRJ_TITLE} =    elyra-test-project
+${PRJ_DESCRIPTION} =    testing Elyra pipeline functionality
+${PV_NAME} =    ods-ci-pv-elyra
+${PV_DESCRIPTION} =    ods-ci-pv-elyra is a PV created to test Elyra in workbenches
+${PV_SIZE} =    2
+${envs_list} =    ${NONE}
+${DC_NAME} =    elyra-s3
+@{IMAGE_LIST}    PyTorch    TensorFlow    TrustyAI
+
+
+*** Test Cases ***
+Verify Pipeline Is Displayed Correctly In Standard Data Science Workbench
+    [Documentation]    Loads an example Elyra pipeline and confirms the Elyra web UI displays it correctly
+    [Tags]    Sanity    Tier1
+    ...       ODS-2197
+    [Setup]    Elyra Pipelines SDS Setup
+    Create Workbench    workbench_title=elyra-sds    workbench_description=Elyra test
+    ...                 prj_title=${PRJ_TITLE}    image_name=Standard Data Science  deployment_size=Small
+    ...                 storage=Persistent  pv_existent=${FALSE}
+    ...                 pv_name=${PV_NAME}  pv_description=${PV_DESCRIPTION}  pv_size=${PV_SIZE}
+    ...                 envs=${envs_list}
+    Start Workbench     workbench_title=elyra-sds
+    Launch And Access Workbench    workbench_title=elyra-sds
+    Clone Git Repository And Open    https://github.com/redhat-rhods-qe/ods-ci-notebooks-main
+    ...    ods-ci-notebooks-main/notebooks/500__jupyterhub/elyra/run-pipelines-on-data-science-pipelines/hello-generic-world.pipeline  # robocop: disable
+    Verify Hello World Pipeline Elements
+
+Verify Pipeline Can Be Submitted And Runs Correctly From Standard Data Science Workbench
+    [Documentation]    Submits an example Elyra pipeline to be run by Data Science Pipelines and
+    ...    Confirms that it runs correctly
+    [Tags]    Sanity    Tier1
+    ...       ODS-2199
+    Set Runtime Image In All Nodes    runtime_image=Datascience with Python 3.9 (UBI9)
+    Run Pipeline
+    Wait Until Page Contains Element    xpath=//a[.="Run Details."]    timeout=30s
+    ${pipeline_run_name} =    Get Pipeline Run Name
+    ${handle} =    Switch To Pipeline Execution Page
+    Verify Successful Pipeline Run Via Project UI   pipeline_run_name=${pipeline_run_name}
+    ...    pipeline_name=hello-generic-world    project_name=${PRJ_TITLE}
+    Switch Window    ${handle}
+    Click Element    //button[.="OK"]
+    [Teardown]    Elyra Pipelines SDS Teardown
+
+Verify Elyra Pipelines In SDS-Based Images
+    [Documentation]    Runs the same Elyra test of the first two test cases in the other images based on SDS
+    ...    (Tensorflow, Pytorch and TrustyAI)
+    [Tags]    Sanity    Tier1
+    ...       ODS-2271
+    [Setup]    Elyra Pipelines SDS Setup
+    FOR    ${img}    IN    @{IMAGE_LIST}
+        Run Elyra Hello World Pipeline Test    ${img}
+    END
+    [Teardown]    Elyra Pipelines SDS Teardown
```

Comment:
why not replace the previous 2 test cases with this one only?

Reply:
I want the SDS image to be tested in isolation since all other images are then based off of it, so if something works in SDS but not in another image we know there must be something wrong in the additional bits of the other img.
Depending on the weigth of Elyra we could also decided to have the SDS img tested in Smoke and the rest in Sanity, or Sanity/T2